libs/application/templates/driving-lessons/src/forms/DrivingLessonsApplication.ts

```diff
@@ -217,10 +220,28 @@ export const DrivingLessonsApplication: Form = buildForm({
       id: 'confirmation',
       name: m.confirmationSection,
       children: [
-        buildIntroductionField({
-          id: 'overview',
-          name: m.overview,
-          introduction: m.overviewIntro,
+        buildMultiField({
+          id: 'submit',
+          name: 'Takk fyrir að sækja um',
+          children: [
+            buildSubmitField({
+              id: 'submit',
+              placement: 'footer',
+              name: 'sick',
+              actions: [
+                {
+                  event: 'SUBMIT',
+                  name: 'Smelltu hér til að senda inn umsókn',
+                  type: 'primary',
+                },
+              ],
+            }),
+            buildIntroductionField({
```

Comment:
Maybe we'll want to re-name `IntroductionField` in the future since it's not just used as an introduction. Something like `buildExplanationField`?

Reply:
yeah, or StaticContentField or something. It is definitely not `IntroductionField` any more :)